docs/chaosiq-on-docker/build-a-custom-docker-image.md

```diff
@@ -0,0 +1,47 @@
+It's recommended to use the  ```chaosiq/chaostoolkit``` Docker image from Docker Hub, which is
+provided by ChaosIQ. However if you want to build a customer Docker image, so you can add your own extensions
+the following can be used as a starter for your own Dockerfile:
+
+```
+FROM python:3.8-alpine
+
+LABEL maintainer="ChaosIQ <contact@chaosiq.io>"
+
+RUN apk add --no-cache --virtual build-deps gcc g++ git libffi-dev linux-headers \
+        python3-dev musl-dev && \
+    pip install --no-cache-dir -q -U pip && \
+    pip install --no-cache-dir chaostoolkit && \
+    pip install --no-cache-dir chaosiq-cloud && \
+    apk del build-deps
+
+ADD . /home/svc
+
+
+
+WORKDIR /home/svc
+
+ENTRYPOINT ["/usr/local/bin/chaos"]
+```
+
+Add the above to a ```Dockerfile```, it can the be built with:
+
+```
+docker build -t chaosiq .
+```
+
+Once you have built your local Docker image you could run an experiment as follows:
+
+```bash
+docker run  -e "ENDPOINT_URL=https://httpstat.us/200?sleep=2000" \
+-v `pwd`/ctk:/home/svc chaosiq run  \
```

Comment:
I would consider using our own official image: chaosiq/chaostoolkit (for all examples)

Reply:
I'm also confused, to show how to run an experimentation here, if that is to show how to build a custom docker image. wdyt ?

Reply:
@dmartin35 - I have changed it and explained they can use their custom image in place of chaosiq/chaostoolkit in the other parts